src/components/ContextMenu.scss

```diff
@@ -73,4 +73,19 @@
   .context-menu-option:focus {
     z-index: 1;
   }
+
+  @media #{$media-query} {
+    .context-menu-option {
+      display: block;
+      margin-inline-end: 0;
+
+      div:nth-child(1) {
```

Comment:
that first child has a separate class (used to set the display: none)

so it's a good choice to only have on css-selector ``.context-menu-option__shortcut`` to reference that element